### DISCUSSION TITLE
Space + R renaming and then holding backspace now closes the renamer. Whyyyyyy

### DISCUSSION THREAD

**Discussion by EriKWDev** [helix-editor/helix#10042] [General] [answered]
I want to rename a variable. Previously I could hit space + R and hold backspace and then type the new name. Now, if I hold backspace for just one character too much the renamer closes.

I really don't think this is a good behaviour even if it is more concistent with other places in the editor. Don't know when this was introduced but it makes me sad

**Comment by kirawi** [comment, marked as answer]
See https://github.com/helix-editor/helix/pull/10005

**Reply by EriKWDev** [reply]
huzza!
